### DISCUSSION TITLE
Autocomplete multiple commands in the terminal emulator

### DISCUSSION THREAD

**Discussion by Bohne13** [Vanilla-OS/live-iso#385] [General]
I'm using "Ptyxis" as daily terminal emulator, but the 'problem' is also present on "Black Box" as far as I know.

If I want to type for example:
`flatpak list --app`
an all previous distros I tiped `flatpak lis` and hit the tab-key for autocomplete, this worked even for longer commands with three or four commands and flags.

- Why does it work on Vanilla OS only for the first command?
- Is there a way to enable autocomplete for all possible command options?

**Comment by taukakao** [comment]
Try enabling the login shell in Ptyxis. This works with distrobox, not sure if it works with apx tho.

And to answer the question, it is an apx bug.

**Reply by Bohne13** [reply]
Thanks for the answer.
In "Black Box" and Ptyxis the login-shell is already enabled, but the autocomplete does not work in either of them.
So the bug is known in apx and will be fixed sometime, maybee?!?

**Reply by taukakao** [reply]
The bug is currently tracked here <https://github.com/Vanilla-OS/desktop-image/issues/193> though it should maybe be transferred to apx, but that can be done once the actual cause is known. 

And yes, it's known but no one has gotten around to fixing this yet.
